atlassian_jwt_auth/key.py

```diff
@@ -29,11 +29,11 @@ class KeyIdentifier(object):
     """ This class represents a key identifier """
 
     def __init__(self, identifier):
-        self.__key_id = validate_key_identifier(identifier)
+        self._key_id = validate_key_identifier(identifier)
 
     @property
     def key_id(self):
-        return self.__key_id
+        return self._key_id
```

Comment:
This change seems unrelated to this pull request.

Reply:
Sorta. This PR started as some minor cleanup, then spiraled out a bit.

I can pull the minor parts out if they're bugging anybody.